superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -205,6 +214,13 @@ class ChartContainer extends React.Component {
               }
 
               <div className="pull-right">
+                <Timer
```

Comment:
could we add some space between the timer and the action buttons to the right? in the screenshot it looks like the are right next to each other. would be nice to add 10px between and make the timer and the buttons aligned in the middle.